kong/pdk/node.lua

```diff
@@ -147,6 +161,17 @@ local function new(self)
   --       pid = 18005
   --     }
   --   }
+  --   -- if the `kong` use dbless mode, the following will be present:
+  --  lmdb = {
+  --    max_map_size: "131072 KB",
+  --    map_size: "131072 KB",
```

Comment:
What is the difference between map_size and max_map_size from the user's perspective?
Can the two be different? Should the user be aware of both values?

Reply:
Also I have some concerns of the string representation of the size. Will this be difficult to consume in an automated fashion? Should the conversion to text representation done by API instead of the PDK?

Remember PDK is a programmable interface, and integer values seems to be the easiest to work with.

Reply:
I delete the `max_map_size` entry.

Reply:
@dndx  this PDK function has unit arguments that mean the caller can control the output data unit. This behavior was trying to follow this function's existing programming interface. https://github.com/Kong/kong/blob/master/kong/pdk/node.lua#L151